web/src/search/input/interactive/FilterInput.test.tsx

```diff
@@ -54,8 +54,7 @@ describe('FilterInput', () => {
         }
         filterHandler(newFiltersInQuery, `${inputValue}`)
     }
-
-    it('filter input for content filters get auto-quoted', () => {
+    it('filter input for content filters get auto-quoted when not editable', () => {
```

Comment:
It would be nice if to have the common parts of test names be extracted into `describe()` blocks, like this:

```typescript
describe('filter input', () => {
    describe('for content filters', () => {
        it('gets auto-quoted when not editable', () => { ... })
    })
})
```


```suggestion
    it('filter input for content filters gets auto-quoted when not editable', () => {
```